lasagne/tests/test_theano_extensions.py

```diff
@@ -98,3 +98,33 @@ def test_pad(val, width=3, batch_ndim=2):
     X_pad_np = np.pad(X0, pads, mode='constant', constant_values=val)
 
     assert (X_pad_theano == X_pad_np).all()
+
+
+def test_pad_width_per_axis(val=0, batch_ndim=2):
```

Comment:
The `batch_ndim` argument is a bit misleading, because setting it to a different value will only affect `X_pad_theano`, but not `X_pad_np` (i.e., it suggests the test was more flexible than it actually is). I'd just remove it. However, we don't have a test for `batch_ndim != 2` now, do we? Maybe just modify one of the tests for `batch_ndim == 1` as an easy way out?